### ISSUE TITLE
dropping Frames / Stuttering after a while playing a movie

### ISSUE THREAD

**Issue by KOKOtm** [koying/SPMC#272]
Hi,

i have the problem, that stuttering /dropping frames occurs while watching a movie. It suddenly happens, only closing SPMC and starting again solves it. Looks nearly the same like 3:2 pulldown, if the refresh rates are wrong. But i checked that, SPMC switched them correctly. Perhaps buffer problems? Shield is connected via LAN, shouldn't be the problem.

Happens with 16.2.1 and 16.3.0.

**Comment by blondin76** [comment]
Hi,

I'have the same problem with shiled
but i have disable in video accelerator "Mediacodec surface" 
I think is better

**Comment by Alamei** [comment]
Out of curiosity, how frequently does the stutter occur? If it's happening roughly every three-and-a half minutes, then this may be the bug I detailed in an earlier issue here: [23.976 FPS Video Stutters Every Few Minutes](https://github.com/koying/SPMC/issues/203).

For now, I've gone back to using stock Kodi, since it doesn't experience the stutter issue. You might try installing Kodi and playing back the same files you're having issues with, just to see if the issue is a bug with SPMC or an upstream issue from Kodi.

**Comment by KOKOtm** [comment]
No, the problem just starts and then stays until restarting SPMC.

**Comment by poorty** [comment]
Hey guys. Just out of curiosity do you have any external storage attached to your shield? SD card or external HDD. I found no matter what version of kodi/SPMC running it would stutter with external storage attached. There seemed to be a bug with the shield itself not accessing the external storage quickly enough to access kodi/SPMC application and meta data. If you do have external storage try removing and reinstalling SPMC/KODI you should have a problem. 

I've researched that there are apparently external storage devices that do work and don't cause stuttering but haven't been asses to try one myself yet. Just happy that the app is working flawlessly now. :-). 

Anyway hope that helps someone.

**Comment by KOKOtm** [comment]
No, just LAN with NFS shares. No external storage attached.

**Comment by Alamei** [comment]
Same deal here. I only playback via LAN shares, and have no external storage devices connected to the Shield.
